src/applications/personalization/profile-2/tests/components/personal-information/PersonalInformation.edit-email-address.unit.spec.jsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
```

Comment:
well crap. github didn't realize that this file is actually renamed. but I also did a fair amount of refactoring as well.